MapboxNavigation/RouteMapViewController.swift

```diff
@@ -166,78 +197,16 @@ class RouteMapViewController: UIViewController {
     override func viewDidAppear(_ animated: Bool) {
         super.viewDidAppear(animated)
         self.annotatesSpokenInstructions = self.delegate?.mapViewControllerShouldAnnotateSpokenInstructions(self) ?? false
-        showRouteIfNeeded()
-        self.currentLegIndexMapped = self.routeController.routeProgress.legIndex
-        self.currentStepIndexMapped = self.routeController.routeProgress.currentLegProgress.stepIndex
+        self.showRouteIfNeeded()
+        self.currentLegIndexMapped = self.routeController?.routeProgress.legIndex ?? 0
+        self.currentStepIndexMapped = self.routeController?.routeProgress.currentLegProgress.stepIndex ?? 0
     }
 
     override func viewWillDisappear(_ animated: Bool) {
         super.viewWillDisappear(animated)
         self.removeTimer()
     }
 
-    func resumeNotifications() {
```

Comment:
Did anything change with this block or was this simply moved?

Reply:
just moved to private section